### PR TITLE
Test/export

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR does:
1. improve oomcli export integraioin-test
2. resolve: #1055

### CODE REVIEW

File: internal/database/offline/sqlutil/export.go [hunk 1]
```diff
@@ -26,6 +26,49 @@ func Export(ctx context.Context, db *sqlx.DB, opt offline.ExportOpt, backend typ
...
+			entity := features[0].Entity()
+
+			schema := ""
+			// export will failed since mysql will trigger an error when you refer to a temporary table more than one time in a single query
+			// see: https://dev.mysql.com/doc/refman/8.0/en/temporary-table-problems.html
+			if backend == types.BackendMySQL {
+				schema = dbutil.BuildTableSchema(tableName, entity, true, features, []string{entity.Name}, backend)
+			} else {
+				schema = dbutil.BuildTempTableSchema(tableName, entity, true, features, []string{entity.Name}, backend)
+			}
```
Comment (jinghancc) [inline internal/database/offline/sqlutil/export.go:64]: maybe wrap the `if else` statement in the `BuildTempTableSchema` function?

Comment (jinghancc) [reply internal/database/offline/sqlutil/export.go:64]: execute could also be wrapped into `BuildTempTableSchema`

File: internal/database/offline/sqlutil/export.go [hunk 2]
```diff
@@ -26,6 +26,49 @@ func Export(ctx context.Context, db *sqlx.DB, opt offline.ExportOpt, backend typ
...
+	for groupID, snapshotTable := range opt.SnapshotTables {
+		if snapshotTable == types.TempTable {
+			tableName := dbutil.TempTable(fmt.Sprintf("snapshot_%d", groupID))
+			features := opt.Features[groupID]
+			entity := features[0].Entity()
+
+			schema := ""
+			// export will failed since mysql will trigger an error when you refer to a temporary table more than one time in a single query
+			// see: https://dev.mysql.com/doc/refman/8.0/en/temporary-table-problems.html
+			if backend == types.BackendMySQL {
```
Comment (jinghancc) [inline internal/database/offline/sqlutil/export.go:38]: should we drop mysql "temporary" tables?

File: internal/database/offline/sqlutil/export.go (hunk 1 above)
Comment (lianxmfor) [reply internal/database/offline/sqlutil/export.go:64]: I was thinking of putting it in BuiltTempTableSchema to get away from the first site and also to make BuiltTempTableSchema not necessarily a build temporary table anymore.

File: internal/database/offline/sqlutil/export.go (hunk 2 above)
Comment (lianxmfor) [reply internal/database/offline/sqlutil/export.go:38]: I should drop the `temporary` table, but I just didn't think of how to do it. What if I drop it in `sqlxQueryExportResults`?

Comment (lianxmfor) [reply internal/database/offline/sqlutil/export.go:38]: it just occurred to me that I might not need to introduce a temporary table here. I'll just create new normal tables and drop them one by one in `sqlxQueryExportResults`. What do you think?

**Review by jinghancc — APPROVED** [review]
LGTM

**Pull request merged**